src/styles/blocks/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  margin: 0 auto;
```

Comment:
```suggestion
```
Don't use outer spacings for BEM blocks. Use mix of classes. `.container` is a BEM block